xarray/tests/test_formatting.py

```diff
@@ -413,6 +413,21 @@ def test_array_repr_variable(self) -> None:
         with xr.set_options(display_expand_data=False):
             formatting.array_repr(var)
 
+    def test_array_scalar_format(self) -> None:
+        var = xr.DataArray(0)
+        assert var.__format__("") == "<xarray.DataArray ()>\narray(0)"
+        assert var.__format__("d") == "0"
+        assert var.__format__(".2f") == "0.00"
+
+        var = xr.DataArray([0])
+        assert var.__format__("") == (
+            "<xarray.DataArray (dim_0: 1)>\narray([0])"
+            "\nDimensions without coordinates: dim_0"
+        )
```

Comment:
```suggestion
        assert var.__format__("") == var.__repr__()
        assert var.__format__("d") == "0"
        assert var.__format__(".2f") == "0.00"

        var = xr.DataArray([0])
        assert var.__format__("") == var.__repr__()
```
repr style can change over time. I think this should work.

Reply:
Thanks that makes sense!

However I think I would like to simplify this even more and always call `ndarray.__format__` when `DataArray.__format__` is called. This won't be backwards compatible but I think this would be better and more predictable.